source/tests/system/nidmm_driver_api_tests.cpp

```diff
@@ -269,6 +336,67 @@ TEST_F(NiDmmDriverApiTest, ConfigureCurrentSourse_CompletesSuccessfully)
     expect_api_success(response.status());
 }
 
+TEST_F(NiDmmDriverApiTest, ConfiguredCurrentSource_ExportAndReset_ImportsSuccessfully)
+{
+    ConfigureCurrentSource(0.0001);
+    
+    auto exportedConfigurationResponse = ExportConfiguration();
+    Reset();
+
+    ImportCoinfiguration(exportedConfigurationResponse);
+    ValidateCurrentSource(0.0001);
+}
+
+TEST_F(NiDmmDriverApiTest, ConfiguredTrigger_ConfiguresSuccessfully)
+{
+    ::grpc::ClientContext configureTriggerContext;
+    dmm::ConfigureTriggerRequest configureTriggerRequest;
+    configureTriggerRequest.mutable_vi()->set_id(GetSessionId());
+    configureTriggerRequest.set_trigger_source(dmm::TriggerSource::TRIGGER_SOURCE_NIDMM_VAL_SOFTWARE_TRIG);
+    configureTriggerRequest.set_trigger_delay(-1);
+    dmm::ConfigureTriggerResponse configureTriggerResponse;
+    
+    ::grpc::Status configureTriggerStatus = GetStub()->ConfigureTrigger(&configureTriggerContext, configureTriggerRequest, &configureTriggerResponse);
```

Comment:
Remove blank line before this. Blank lines in method body should match the "_" in the test name.